cardano-cli/src/Cardano/CLI/EraBased/Transaction/HashCheck.hs

```diff
@@ -0,0 +1,84 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Cardano.CLI.EraBased.Transaction.HashCheck
+  ( checkCertificateHashes
+  , checkVotingProcedureHashes
+  , checkProposalHashes
+  )
+where
+
+import           Cardano.Api (Certificate (..), ExceptT, firstExceptT)
+import qualified Cardano.Api as L
```

Comment:
```suggestion
import Cardano.Api
```
I think this shouldn't be imported qualified `as L`

Reply:
Addressed in rebase

Reply:
Addressed in rebase